### ISSUE TITLE
Test failure in Structural plasticity

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#7762]
~~~
======================================================================
FAIL: test_execution (structural_mechanics_test_factory.SmallDeformationPlasticityTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philipp/software/Kratos_MasterBranch/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 72, in test_execution
    self.test.RunSolutionLoop()
  File "/home/philipp/software/Kratos_MasterBranch/install/KratosMultiphysics/analysis_stage.py", line 69, in RunSolutionLoop
    self.FinalizeSolutionStep()
  File "/home/philipp/software/Kratos_MasterBranch/install/KratosMultiphysics/analysis_stage.py", line 151, in FinalizeSolutionStep
    process.ExecuteFinalizeSolutionStep()
  File "/home/philipp/software/Kratos_MasterBranch/install/KratosMultiphysics/from_json_check_result_process.py", line 55, in ExecuteFinalizeSolutionStep
    self.assertTrue(self.process.IsCorrectResult(), "Results do not coincide with the JSON reference\n" + self.process.GetErrorMessage())
AssertionError: False is not true : Results do not coincide with the JSON reference
Error checking for variable UNIAXIAL_STRESS in GP 0, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 1, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 2, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 3, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 4, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 5, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 6, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
Error checking for variable UNIAXIAL_STRESS in GP 7, results:
Element 1 2980492!=3000002; rel_tol=1e-06, abs_tol=0.001
~~~

can you Guys please check?
We want to have it all working for the upcoming release
Cheers

**Comment by SergioJimenezReyes** [comment]
Yes, no problem, @AlejandroCornejo and myself are already aware about this and we will solve it as soon as possible.

**Comment by AlejandroCornejo** [comment]
I've created a branch and compiling, I'll do a PR today

**Comment by AlejandroCornejo** [comment]
> I've created a branch and compiling, I'll do a PR today

https://github.com/KratosMultiphysics/Kratos/pull/7771

**Comment by philbucher** [comment]
fixed by #7779, thx @AlejandroCornejo !
